init/src/lib.rs

```diff
@@ -85,17 +123,29 @@ pub fn execute(path: &Path) -> anyhow::Result<()> {
     let (signer, storage) = keys::storage(&profile, sock)?;
     let identity = person::local(&storage)?;
 
-    let head: String = repo
-        .head()
-        .ok()
-        .and_then(|head| head.shorthand().map(|h| h.to_owned()))
-        .ok_or_else(|| anyhow!("error: repository head does not point to any commits"))?;
+    // If input is not passed as CLI argument, get user input.
+    if name.is_empty() {
+        let cwd = std::env::current_dir()?;
+        let path = cwd.as_path();
+        name = path.file_name().map(|f| f.to_string_lossy().to_string());
+        name = term::text_input("Name", name)?;
+    }
 
-    git::check_version()?;
+    if description.is_empty() {
+        description = term::text_input("Description", None)?;
+    }
+    if branch.is_empty() {
+        let head: String = repo
+            .head()
+            .ok()
+            .and_then(|head| head.shorthand().map(|h| h.to_owned()))
+            .ok_or_else(|| anyhow!("error: repository head does not point to any commits"))?;
+        
+        git::check_version()?;
```

Comment:
We should always check the git version, so maybe move this out above all the `if`